src/Sylius/Bundle/AddressingBundle/Form/Type/CountryCodeChoiceType.php

```diff
@@ -61,10 +87,11 @@ private function getCountryCodes(array $countries)
 
         /* @var CountryInterface $country */
         foreach ($countries as $country) {
-            $countryCodes[$country->getCode()] = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryName = Intl::getRegionBundle()->getCountryName($country->getCode());
+            $countryCodes[$countryName] = $country->getCode();
```

Comment:
So in Symfony >=3.0 you must use array keys as values and there is no option to reverse that?